src/utils/index.ts

```diff
@@ -38,10 +38,17 @@ export function getErrorMessage(value: unknown): string {
         return value.message;
     }
 
+    // If the thrown error is an instance of Error, React Router will convert it to a string.
```

Comment:
Just talked to Oleh, this first line of the comment is not quite correct. This only applies to errors wrapped with React Router's private [ErrorResponseImpl](https://github.com/remix-run/react-router/blob/9c32cfa3529071373ff306d7711c34f7052dfc78/packages/router/utils.ts#L1670) object.

I'll push an updated comment to this branch.